compiler/src/iree/compiler/Codegen/Dialect/UKernelOps.cpp

```diff
@@ -65,19 +69,22 @@ static FailureOr<func::CallOp> createFunctionCall(RewriterBase &rewriter,
   return rewriter.create<func::CallOp>(loc, fnDecl, callOperands);
 }
 
-//===---------------------------------------------------------------------===//
-// UKernelGenericOp
-//===---------------------------------------------------------------------===//
-
-std::pair<int64_t, int64_t> UKernelGenericOp::getDpsInitsPositionRange() {
-  auto [pos, size] = getODSOperandIndexAndLength(1);
-  return {static_cast<int64_t>(pos), static_cast<int64_t>(pos + size)};
-}
+/// Enum to describe what function arguments to pass to the ukernel target
+/// function for each memref operand of the ukernel op.
+enum class BufferPassingStyle {
+  // Pass only the base buffer and offset, the bare minimum for the callee to be
+  // able to compute the effective buffer address.
+  kBufferAndOffset,
+  // Also pass the N-1 first strides for a rank-N memref, i.e. all but the
+  // inner-most stride.
+  kBufferAndOffsetAndNm1Strides
```

Comment:
Please add a TODO (and if you feel like it file a bug as well and assign to me, but TODO with my name here is enough) to drop this to not use `OffsetAndNm1Strides`, but instead use `OffsetSizesAndStrides`.